pkg/solver/controller.go

```diff
@@ -332,12 +332,23 @@ func (controller *SolverController) addResourceOffer(resourceOffer data.Resource
 		return nil, fmt.Errorf("failed to retrieve ETH balance for resource provider: %v", err)
 	}
 	// Convert InstructionPrice from ETH to Wei
-	requiredBalanceWei := web3.EtherToWei(float64(resourceOffer.DefaultPricing.InstructionPrice))
+	requiredBalanceWei := web3.EtherToWei(0.0006) // 0.0006 based on the required balance for a job
```

Comment:
[nit] Make the `0.0006` a variable so it's easier to track and change when we need to

Reply:
resolved on the latest commit.